File: Day-8/program1.cpp
#include <iostream>
using namespace std;
int main() 
{//opening main
    int r;//initialising variables
    cout<<"enter number of rows";
    cin>>r;//Taking input from user
    for (int i = 1; i <= r; i++)
    {//OPENING FOR LOOP
        for (int j = 1; j <= i; j++)
         {//opening for loop
            cout << "* ";
        }//closing for loop
        cout<<endl;
    }//CLOSING FOR LOOP
    return 0;
}//closing main

File: Day-13/program1.java
import java.util.Scanner;
public class program1
{//opening class
    public static void main(String[] args) 
    {//opening main
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter size of array: ");
        int n = sc.nextInt();//initialising variables
        int ar[] = new int[n];
        System.out.println("Enter array elements=");
        for(int i=0;i<n;i++)
             {//opening for loop
            ar[i] = sc.nextInt();
             }//closing for loop
        System.out.println("Array elements are=");
        for(int i=0;i<n;i++) 
            {//opening for loop
            System.out.println(ar[i]+" ");
            }//closing for loop
        sc.close();
    }//closing main
}//closing class